guava/src/com/google/common/util/concurrent/ServiceManager.java

```diff
@@ -661,6 +703,35 @@ public Long apply(Entry<Service, Long> input) {
       return ImmutableMap.copyOf(loadTimes);
     }
 
+    ImmutableMap<Service, Long> stopTimes() {
+      List<Entry<Service, Long>> stopTimes;
+      monitor.enter();
+      try {
+          stopTimes = Lists.newArrayListWithCapacity(stopTimers.size());
```

Comment:
+2 for statement indentation here to match rest of file.

Reply:
done